DOCS/man/input.rst

```diff
@@ -1404,7 +1445,7 @@ Screenshot Commands
     On success, returns a ``mpv_node`` with a ``filename`` field set to the
     saved screenshot location.
 
-``screenshot-to-file <filename> [<flags>]``
+``screenshot-to-file [<filename> [<flags>]]``
     Take a screenshot and save it to a given file. The format of the file will
     be guessed by the extension (and ``--screenshot-format`` is ignored - the
     behavior when the extension is missing or unknown is arbitrary).
```

Comment:
To Add `if arguments are empty, file dialog will be shown to select target location` here too?